src/config/config.js

```diff
@@ -1,7 +1,8 @@
 import keys from './keys';
 
 export default {
-	api: process.env.VUE_APP_API_URL || 'http://localhost:3000',
+	api:
+		process.env.VUE_APP_API_URL || 'https://frontend-social-api.herokuapp.com',
```

Comment:
can you please revert this change as it is needed for running the app code on localhost?

Reply:
Reverted changes